src/sbp.h

```diff
@@ -13,9 +13,9 @@
 #ifndef SWIFTNAV_SBP_H
 #define SWIFTNAV_SBP_H
 
-#include <libswiftnav/common.h>
-#include <libswiftnav/sbp.h>
-#include <libswiftnav/sbp_messages.h>
+#include <libsbp/common.h>
+#include <libsbp/sbp.h>
+#include <libsbp/sbp_messages.h>
```

Comment:
.... so `libsbp/sbp_messages.h` is actually the legacy header moved from libswiftnav to libsbp containing the messages that are now split up across `libsbp/standard.h` and `libsbp/navigation.h`. `libsbp/sbp_messages.h` needs to be removed from from the libsbp c includes.